### PR TITLE
Added Press, shakeY and Refactory in styles in transform-origin

### PR DESCRIPTION
**Pull request by lucasferreiralimax**
Add new animation Press and shakeY
Refactory shake name for shakeX
* press.css
* shakeX.css
* shakeY.css

Refactory styles in transform-origin
* rotateIn.css
* rotateInDownLeft.css
* rotateInDownRight.css
* rotateInUpLeft.css
* rotateInUpRight.css
* rotateOut.css
* rotateOutDownLeft.css
* rotateOutDownRight.css
* rotateOutUpLeft.css
* rotateOutUpRight.css
* hinge.css
* zoomOutDown.css
* zoomOutLeft.css
* zoomOutRight.css
* zoomOutUp.css

Update animate-config and files of project

[Preview Codepen](https://codepen.io/lucaslimax/pen/yLBdKjJ) 👍

### CODE REVIEW

**Comment by WarenGonzaga** [comment]
Hello there! For me, I would accept the Press animation and for the ShakeX and Y can you please convince us more why we need this on v4? It looks like the same as Shake Animation.

**Comment by lucasferreiralimax** [comment]
Are opposite animations, press is different from current pulse and shake only horizontally with shake axes do you have the opposite animation.

can see new animations with opposite.

**Comment by WarenGonzaga** [comment]
@lucasferreiralimax that's cool! Alright... you already convince me... and we will add this to the v4 of animate.css ;) thanks for the awesome PR.

**Comment by lucasferreiralimax** [comment]
Thanks @WarenGonzaga, i want has collaboration forever more and more. 😃

**Comment by WarenGonzaga** [comment]
> Thanks @WarenGonzaga, i want has collaboration forever more and more. 😃
> 
> I improvement the transform-origin also inside PR

Well, that's cool! Thanks for doing that... @eltonmesquita kindly review the code for final checking.. thanks!

**Comment by WarenGonzaga** [comment]
@lucasferreiralimax I'm gonna review your code and will be added to the current development.

**Comment by eltonmesquita** [comment]
Thanks for the PR. I think we can merge the refactor and the shake animations, but I don't think that the press animation is a good fit for the library. Please review the [contributing guide](https://github.com/daneden/animate.css/blob/master/CONTRIBUTING.md).

Before merging, could you please split the refactor from the animations proposal, removing the press animation in the process? Also, it would help a lot if you could PR it into the dev branch and not the master as it's more up to date and will become the v4 soon.

File: animate-config.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
     "bounce": true,
     "flash": true,
     "pulse": true,
+    "press": true,
```
Comment (eltonmesquita) [inline animate-config.json:6]: This file is no more on the dev branch. Make sure you update https://github.com/daneden/animate.css/blob/dev/source/animate.css instead.

File: source/rotating_entrances/rotateIn.css [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
   to {
-    transform-origin: center;
     transform: translate3d(0, 0, 0);
     opacity: 1;
   }
 }
 
 .rotateIn {
   animation-name: rotateIn;
+  transform-origin: center;
```
Comment (eltonmesquita) [inline source/rotating_entrances/rotateIn.css:15]: Please, split the refactoring into other PR.

**Comment by eltonmesquita** [comment]
I've cherry-picked `shakeX`, `shakeY` and the `transform-origin` refactor into the dev branch, so I'm closing this.

Thanks for the PR!

**Pull request closed without merging**